seatunnel-connectors-v2/connector-redis/src/main/java/org/apache/seatunnel/connectors/seatunnel/redis/config/RedisParameters.java

```diff
@@ -89,6 +90,9 @@ public void buildWithConfig(Config config) {
         if (config.hasPath(RedisConfig.KEY_PATTERN.key())) {
             this.keysPattern = config.getString(RedisConfig.KEY_PATTERN.key());
         }
+        if (config.hasPath(RedisConfig.EXPIRE.key())) {
+            this.expire = config.getLong(RedisConfig.EXPIRE.key());
```

Comment:
It is suggest to use `EXPIRE.defaultvalue()`

Reply:
Default values have been added. In Redis, the default value of the EXPIRE command is -1, and keys are not automatically expired by default.